### ISSUE TITLE
babel 7 support

### ISSUE THREAD

**Issue by kovenliao** [michalkvasnicak/babel-plugin-css-modules-transform#72]
I am using babel7.0.0-beta.34 but got below error

`Module build failed: TypeError: Invalid attempt to destructure non-iterable instance`

The error was throw from this line
https://github.com/michalkvasnicak/babel-plugin-css-modules-transform/blob/master/src/index.js#L103
```
thisPluginOptions = options.plugins.filter(
      ([plugin]) => plugin.manipulateOptions === pluginApi.manipulateOptions
)[0][1];
```

The `options.plugins` object was changed in Babel7, can we consider fix it like this?
https://github.com/michalkvasnicak/babel-plugin-css-modules-transform/compare/master...kovenliao:fix?expand=1

**Comment by louisgjohnson** [comment]
yes please ^^

**Comment by kovenliao** [comment]
please review
https://github.com/michalkvasnicak/babel-plugin-css-modules-transform/pull/73/files

**Comment by saiichihashimoto** [comment]
This package is my one blocker to using babel 7 and #73 fixes it. Any ETA on when it will be reviewed?

**Comment by michalkvasnicak** [comment]
Thank you very much, released in 1.4.0. Sorry that it took so long, holidays in Slovakia.
